src/scripts/createMarkdown.sh

```diff
@@ -5,42 +5,42 @@ set -e
 
 # Ensure two arguments are provided
 if [ "$#" -ne 2 ]; then
-   echo "Usage: $0 <filename> <relative-path>"
-   exit 1
+  echo "Usage: $0 <filename> <relative-path>"
+  exit 1
 fi
 
 # Define arguments
 FILENAME="$1.md"
 RELATIVE_PATH="$2"
-BASE_PATH="public/content/platforms"
+BASE_PATH="public/content/documentation"
 
 # Construct the full path
 FULL_PATH="${BASE_PATH}/${RELATIVE_PATH}"
 
 # Check if the directory exists, if not create it
 if [ ! -d "$FULL_PATH" ]; then
-   mkdir -p "$FULL_PATH"
-   echo "Created directory: $FULL_PATH"
+  mkdir -p "$FULL_PATH"
+  echo "Created directory: $FULL_PATH"
 fi
 
 # Create the markdown file if it doesn't already exist
 FILE_PATH="${FULL_PATH}/${FILENAME}"
 if [ -f "$FILE_PATH" ]; then
-   echo "Error: File '$FILE_PATH' already exists!"
-   exit 1
+  echo "Error: File '$FILE_PATH' already exists!"
+  exit 1
 fi
 
 # Create the markdown file with a template
 cat >"$FILE_PATH" <<"EOF"
-# General Notes
+## General Notes
 > This file was generated using the createMarkdown script.
 
 This section provides general information about this component, including guidelines and best practices.
 
-# Condensed
+## Condensed
```

Comment:
For each section (condensed, gherkin, developer notes, ios dev, android dev), I would copy/paste the criteria from something already existing, such as the web header gherkin/condensed and the native button ios and android dev notes so that it's as little change as we are going to have to make later on